### PR TITLE
Fix the alerts about conditional tasks

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/41263

## Description
Fixed a n issue with alerts about conditional tasks in playbook

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * Added a validation to ensure reputation command's default argument is set as an array input.
 * Added the `--fail-duplicates` flag for the **merge-id-set** command which will fail the command if duplicates are found.
 * Added the `--fail-duplicates` flag for the **create-id-set** command which will fail the command if duplicates are found.
+* Fixed an issue in **Validate** in playbooks with conditional tasks
```
Comment (reutshal) [inline CHANGELOG.md:10]:
```suggestion
* Fixed an issue in **Validate** command where playbooks conditional tasks were mishandeled.
```

File: demisto_sdk/commands/common/hook_validations/playbook.py [hunk 2]
```diff
@@ -177,11 +177,18 @@ def is_ask_condition_branches_handled(self, task: Dict) -> bool:
         # ADD all replyOptions to unhandled_reply_options (UPPER)
         unhandled_reply_options = set(map(str.upper, task.get('message', {}).get('replyOptions', [])))
 
+        # Rename the keys in dictionary to upper case
+        next_tasks_upper = {k.upper(): v for k, v in next_tasks.items()}
+
+        # Rename the dictionary keys from 'True Positive\False Positive' to 'YES\NO'
+        next_tasks_upper['YES'] = next_tasks_upper.pop('TRUE POSITIVE', next_tasks_upper.get('YES'))
+        next_tasks_upper['NO'] = next_tasks_upper.pop('FALSE POSITIVE', next_tasks_upper.get('NO'))
```
Comment (reutshal) [inline demisto_sdk/commands/common/hook_validations/playbook.py:185]: What was the problem? 
Some of the tasks had 'TRUE/FALSE POSITIVE' while the others had 'YES/NO'?

File: demisto_sdk/commands/common/tests/playbook_test.py [hunk 3]
```diff
@@ -109,6 +109,12 @@ class TestPlaybookValidator:
                                                          {'1': {'type': 'condition',
                                                                 'message': {'replyOptions': ['yes']},
                                                                 'nexttasks': {'YES': ['1']}}}}
+    CONDITIONAL_ASK_EXISTS_WITH_TRUE_AND_FALSE_POSITIVE = {"id": "Intezer - scan host", "version": -1,
```
Comment (reutshal) [inline demisto_sdk/commands/common/tests/playbook_test.py:112]:
```suggestion
    CONDITIONAL_TASK_EXISTS_WITH_TRUE_AND_FALSE_POSITIVE = {"id": "Intezer - scan host", "version": -1,
```

**Review by reutshal — APPROVED** [review]
Nice, please see my comments

File: demisto_sdk/commands/common/hook_validations/playbook.py (hunk 2 above)
Comment (MosheEichler) [reply demisto_sdk/commands/common/hook_validations/playbook.py:185]: Exactly

**Pull request merged**